interface/patient_file/summary/pnotes_full.php

```diff
@@ -478,6 +479,9 @@ function show_div(name){
         echo "  <td class='notecell' id='".htmlspecialchars($row_note_id, ENT_QUOTES)."'>\n";
         echo "   $body";
         echo "  </td>\n";
+        echo "  <td class='notecell' id='".htmlspecialchars($row_note_id, ENT_QUOTES)."'>\n";
+        echo $iter['message_status'];
```

Comment:
Proper way to convert items to translated titles is the following:
``````
echo getListItemTitle("message_status", $iter['message_status']);
``````

Reply:
@bradymiller 
done